xarray/namedarray/core.py

```diff
@@ -625,7 +636,7 @@ def __dask_postpersist__(
     ]:
         if is_duck_dask_array(self._data):
             a: tuple[PostPersistCallable[Any], tuple[Any, ...]]
-            a = self._data.__dask_postpersist__()  # type: ignore[no-untyped-call]
+            a = self._data.__dask_postpersist__()  # type: ignore
```

Comment:
These should be raising errors now since we require that error type be specified for `type: ignore`